stack/core/src/main/java/org/apache/usergrid/corepersistence/ApplicationIdCacheFactory.java

```diff
@@ -0,0 +1,43 @@
+/*
+ *
```

Comment:
Since there's only a single instance of the cache, can we just use a Guice Provider?